Source/WebCore/html/NumberInputType.cpp

```diff
@@ -165,8 +156,8 @@ StepRange NumberInputType::createStepRange(AnyStepHandling anyStepHandling) cons
         }
         return defaultValue;
     };
-    Decimal minimum = extractBound(minAttr, -floatMax);
-    Decimal maximum = extractBound(maxAttr, floatMax);
+    Decimal minimum = extractBound(minAttr, -doubleMax);
```

Comment:
Can we use double min here instead of -max?

Reply:
It gives build error and it seems that we don't have any "doubleMin" in Decimal.cpp.